lib/ansible/modules/web_infrastructure/ansible_tower/tower_inventory.py

```diff
@@ -102,7 +113,8 @@ def main():
 
             if state == 'present':
                 result = inventory.modify(name=name, organization=org['id'], variables=variables,
-                                          description=description, create_on_missing=True)
+                                          description=description, kind=kind, host_filter=host_filter,
```

Comment:
@ryanpetrello do you think there may be any problems with past versions by passing kind & host_filter? I would say probably not, as long as the tower-cli version was up-to-date, it may send an extra field, but that would be ignored by the API.